templates/nextjs/vitest.config.mts

```diff
@@ -0,0 +1,5 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  esbuild: { target: "es2022" },
```

Comment:
Needed for `using` keyword

Reply:
The issue https://github.com/vitest-dev/vitest/issues/4183 has apparently been resolved and released with [`vitest v2.0`](https://github.com/vitest-dev/vitest/releases/tag/v2.0.0):
> Transpile esnext to node18 to support newest JS and TS features - by @sheremet-va in https://github.com/vitest-dev/vitest/pull/4409 [(8f65a)](https://github.com/vitest-dev/vitest/commit/8f65ae906)

You might want to update the template's `vitest` to 2.0 as well?

Reply:
Ran into some other type issues when upgrading to `v2`, could we save the update for now and keep the config file?